code/modules/research/destructive_analyzer.dm

```diff
@@ -121,6 +121,11 @@ Note: Must be placed within 3 tiles of the R&D Console
 		attackby(I, L)
 
 
+/obj/machinery/r_n_d/destructive_analyzer/kick_act(mob/living/carbon/human/H)
```

Comment:
Does this let you complete research with no power?

Reply:
Surely not, that would mean gremlins can do research without power th- oh goddamit gremlins can do research without power

Well, only one item anyways, because you can't put another one in while the power is out